src/meetingsession/MeetingSessionStatus.ts

```diff
@@ -172,6 +178,10 @@ export default class MeetingSessionStatus {
         return new MeetingSessionStatus(MeetingSessionStatusCode.VideoCallSwitchToViewOnly);
       case 509:
         return new MeetingSessionStatus(MeetingSessionStatusCode.VideoCallAtSourceCapacity);
+      case 403:
+        return new MeetingSessionStatus(MeetingSessionStatusCode.AudioAuthenticationRejected);
```

Comment:
Would be great to have some comment here explaining why these status are both in signaling and audio

Reply:
Sure. Is more an artifact of the fact that most of these should not have been prefixed by the media type.